CHANGELOG.md

```diff
@@ -902,6 +902,8 @@ should use 4.0.1-alpha.0 for testing.
 #### web3-utils
 
 -   Export a new function `uuidV4` that generates a random v4 Uuid (#5373).
+-   Enable passing a starting number, to increment based on it, for the Json Rpc Request `id` (#5309).
+-   Export a new function `isPromise` that generates a random v4 Uuid (#5309).
```

Comment:
I don't believe this is correct

```
Export a new function `isPromise` that generates a random v4 Uuid (#5309).
```

Also don't we link the PR numbers in the `CHANGELOG` and not issues numbers?

Reply:
Thanks @spacesailor24 
I thought it was the issue id that we put in. However, now I checked the CHANGELOG of React and Angular. They use the MR id: 

- https://github.com/facebook/react/blob/main/CHANGELOG.md
- https://github.com/angular/angular/blob/main/CHANGELOG.md

So, I updated the ids now for this MR.